packages/node-http-handler/src/node-http-handler.ts

```diff
@@ -151,6 +152,16 @@ export class NodeHttpHandler implements HttpHandler {
         };
       }
 
+      const httpAgent = nodeHttpsOptions.agent;
```

Comment:
Add link to upstream bug report
```suggestion
      // Workaround for bug in Node.js https://github.com/nodejs/node/issues/47137
      const httpAgent = nodeHttpsOptions.agent;
```